packages/side-nav/src/SideNavItem.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { MenuItem, MenuItemProps } from '@leafygreen-ui/menu';
+import { LEFT_RIGHT_OFFSET } from './styles';
+
+// style overrides for menu item
+const containerStyleOverrides = css`
+  padding: 10px ${LEFT_RIGHT_OFFSET}px 10px ${LEFT_RIGHT_OFFSET}px;
+  min-height: 0;
+  border-radius: 5px;
+
+  &:before {
+    content: normal;
+  }
+
+  &:focus:active {
```

Comment:
It would be nice to implement this focus state conditionally with the [useUsingKeyboardContext](https://github.com/mongodb/leafygreen-ui/tree/master/packages/leafygreen-provider#useusingkeyboardcontext) hook from the leafygreen-provider package.

This way applications with the provider can benefit from better focus state handling (and it gracefully degrades when the provider isn't a parent).